packages/twenty-front/src/modules/settings/integrations/components/SettingsIntegrationDatabaseConnectionForm.tsx

```diff
@@ -7,7 +7,7 @@ import { TextInput } from '@/ui/input/components/TextInput';
 export const settingsIntegrationPostgreSQLConnectionFormSchema = z.object({
   dbname: z.string().min(1),
   host: z.string().min(1),
-  port: z.number().positive(),
+  port: z.string().min(1),
```

Comment:
Maybe we could let this as a number and transform it to a string after form validation?

Reply:
Not sure, but could perhaps this work?
`port: z.preprocess((val) => parseInt(val), z.number().int().positive()),`
See : https://zod.dev/?id=preprocess
Otherwise you can let this as a string :)